snippets/mssql.json

```diff
@@ -294,5 +294,54 @@
 			"GO"
 		],
 		"description": "Get Space Used by Tables"
+	},
+
+	"Lists all procedures by schema": {
+		"prefix": "sqlListProcedures",
+		"body": [
+			"-- Lists all procedures by schema ${1:ShemaName}",
+            "SELECT",
```

Comment:
Minor: it looks like the indentation is inconsistent. If possible update as it helps with file readability.